packages/@aws-cdk/cloudformation-include/test/valid-templates.test.ts

```diff
@@ -414,6 +414,69 @@ describe('CDK Include', () => {
       includeTestTemplate(stack, 'non-existent-resource-type.json');
     }).toThrow(/Unrecognized CloudFormation resource type: 'AWS::FakeService::DoesNotExist'/);
   });
+
+  test("can ingest a template that contains outputs and modify them", () => {
+    const cfnTemplate = includeTestTemplate(stack, 'outputs-with-references.json');
+    const output = cfnTemplate.getOutput('Output1');
+
+    const ifCond = core.Fn.conditionIf("AlwaysFalseCond", 'AWS::NoValue', 'AWS::NoValue');
+
+    output.value = 'a mutated value';
```

Comment:
What if we nulled something out, and assert it's gone from the resulting template? That seems like a valuable test.

Reply:
like `output.description = undefined;` and testing that it's not in the resulting template?

Reply:
Yep.